_pkgdown.yml

```diff
@@ -2,3 +2,95 @@ navbar:
   right:
     - icon: fa-github
       href: https://github.com/NCEAS/arcticdatautils
+reference:
+
+-title: Functions
+
+- subtitle: Access functions
+  desc: Functions to change who can access the package
+  contents:
+  - starts_with("add_")
+
+- subtitle: DataOne functions
+  desc: Helper functions for the dataone package
+  contents:
+  - 'is_token_set'
+  - 'get_token'
+  - 'is_token_expired'
+  - 'get_mn_base_url'
+  - 'is_authorized'
+
+- subtitle: NetCDF Attribute functions
+  desc: Get a data.frame of attributes from a NetCDF object
+  contents:
+  - 'get_ncdf4_attributes'
+  - 'get_token'
+  - 'is_token_expired'
+  - 'get_mn_base_url'
+  - 'is_authorized'
+
+- subtitle: Sysmeta Function
+  desc: Function to access the sysmeta of the package
+  contents:
+  - 'get_all_sysmeta'
+
+
+- subtitle: Editing Functions
+  desc: Functions to edit a package
+  contents:
+  - 'publish_object'
+  - 'update_object'
+  - 'publish_update'
+  - 'create_resource_map'
+  - 'update_resource_map '
+  - 'set_file_name'
+  - 'reformat_file_name'
+
+- subtitle: Helper Functions
+  desc: Functions to help test a package
+  contents:
+  -  starts_with("create_")
+  - 'get_orcid_name'
+  - 'get_orcid_email'
+
+- subtitle: EML Functions
+  desc: Functions for creating and changing EML metadata
```

Comment:
also probably good candidates to be moved to the EML package. `eml_get_simple` has been mentioned to be added : https://github.com/ropensci/EML/issues/303 but I don't think we ever got around to doing that